[2021-01-14 22:49:49,605] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: MarketVol.download_apple_price 2021-01-14T06:00:00+00:00 [queued]>
[2021-01-14 22:49:49,611] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: MarketVol.download_apple_price 2021-01-14T06:00:00+00:00 [queued]>
[2021-01-14 22:49:49,611] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2021-01-14 22:49:49,611] {taskinstance.py:1018} INFO - Starting attempt 3 of 4
[2021-01-14 22:49:49,611] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2021-01-14 22:49:49,617] {taskinstance.py:1038} INFO - Executing <Task(PythonOperator): download_apple_price> on 2021-01-14T06:00:00+00:00
[2021-01-14 22:49:49,620] {standard_task_runner.py:51} INFO - Started process 73292 to run task
[2021-01-14 22:49:49,627] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'MarketVol', 'download_apple_price', '2021-01-14T06:00:00+00:00', '--job-id', '139', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/stock_dags.py', '--cfg-path', '/var/folders/hy/5f75_vt15hb_bfkv9vvnx5sw0000gn/T/tmp_26j833y']
[2021-01-14 22:49:49,628] {standard_task_runner.py:76} INFO - Job 139: Subtask download_apple_price
[2021-01-14 22:49:49,665] {logging_mixin.py:103} INFO - Running <TaskInstance: MarketVol.download_apple_price 2021-01-14T06:00:00+00:00 [running]> on host 1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa
[2021-01-14 22:49:49,687] {taskinstance.py:1232} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=MarketVol
AIRFLOW_CTX_TASK_ID=download_apple_price
AIRFLOW_CTX_EXECUTION_DATE=2021-01-14T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-01-14T06:00:00+00:00
[2021-01-14 22:49:49,688] {logging_mixin.py:103} INFO - 2021-01-14
[2021-01-14 22:49:49,689] {logging_mixin.py:103} INFO - 2021-01-15
[2021-01-14 22:49:49,890] {logging_mixin.py:103} INFO - [*********************100%***********************]  1 of 1 completed
[2021-01-14 22:49:49,891] {logging_mixin.py:103} INFO - 
[2021-01-14 22:49:49,893] {taskinstance.py:1396} ERROR - [Errno 2] No such file or directory: '/Users/chowli/Documents/Springboard_DE/Projects/mini-projects/Airflow/marketvol/tmp/data/2021-01-14/AAPL.csv'
Traceback (most recent call last):
  File "/Users/chowli/.local/share/virtualenvs/marketvol-WbDfdy4N/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1086, in _run_raw_task
    self._prepare_and_execute_task_with_callbacks(context, task)
  File "/Users/chowli/.local/share/virtualenvs/marketvol-WbDfdy4N/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1260, in _prepare_and_execute_task_with_callbacks
    result = self._execute_task(context, task_copy)
  File "/Users/chowli/.local/share/virtualenvs/marketvol-WbDfdy4N/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1300, in _execute_task
    result = task_copy.execute(context=context)
  File "/Users/chowli/.local/share/virtualenvs/marketvol-WbDfdy4N/lib/python3.7/site-packages/airflow/operators/python.py", line 117, in execute
    return_value = self.execute_callable()
  File "/Users/chowli/.local/share/virtualenvs/marketvol-WbDfdy4N/lib/python3.7/site-packages/airflow/operators/python.py", line 128, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/Users/chowli/Documents/Springboard_DE/Projects/mini-projects/Airflow/marketvol/airflow/dags/stock_dags.py", line 32, in fetch_price
    "{}/tmp/data/{}/{}.csv".format(os.getcwd(), start_date, kwargs['ticker']))
  File "/Users/chowli/.local/share/virtualenvs/marketvol-WbDfdy4N/lib/python3.7/site-packages/pandas/core/generic.py", line 3400, in to_csv
    storage_options=storage_options,
  File "/Users/chowli/.local/share/virtualenvs/marketvol-WbDfdy4N/lib/python3.7/site-packages/pandas/io/formats/format.py", line 1083, in to_csv
    csv_formatter.save()
  File "/Users/chowli/.local/share/virtualenvs/marketvol-WbDfdy4N/lib/python3.7/site-packages/pandas/io/formats/csvs.py", line 234, in save
    storage_options=self.storage_options,
  File "/Users/chowli/.local/share/virtualenvs/marketvol-WbDfdy4N/lib/python3.7/site-packages/pandas/io/common.py", line 644, in get_handle
    newline="",
FileNotFoundError: [Errno 2] No such file or directory: '/Users/chowli/Documents/Springboard_DE/Projects/mini-projects/Airflow/marketvol/tmp/data/2021-01-14/AAPL.csv'
[2021-01-14 22:49:49,899] {taskinstance.py:1440} INFO - Marking task as UP_FOR_RETRY. dag_id=MarketVol, task_id=download_apple_price, execution_date=20210114T060000, start_date=20210115T064949, end_date=20210115T064949
[2021-01-14 22:49:49,936] {local_task_job.py:118} INFO - Task exited with return code 1
